sinon/lib/spy.py

```diff
@@ -6,6 +6,8 @@
 from .util import CollectionHandler as uch
 from .base import SinonBase
 from .matcher import SinonMatcher, Matcher
+from .call import Call
```

Comment:
I don't find any `Call()` declaration here, do you need to use `Call()` in spy?

Reply:
I originally put `Call()` in `spy`, but then moved the call to `wrapper`. I left this import line by accident. Good catch!